daml-script/runner/BUILD.bazel

```diff
@@ -31,6 +31,8 @@ da_scala_library(
         [
             "//canton:community_admin-api_proto_scala",
             "//canton:community_admin-api-src",
+            # Needed just for the topology read service, might make script a bit large
+            "//canton:community_base_proto_scala",
```

Comment:
Need to decide if this is acceptable, as it'll be included in the normal daml-script runner binary.

Reply:
I don't see how this would be a problem considering we already have `//canton:community_admin-api_proto_scala`

Reply:
I believe base is quite a bit larger than admin

Reply:
having checked, with this change `//daml-script/runner:daml-script-binary_deploy.jar` goes from 108MB to 114MB, so about ~5.5% bigger :shrug: